src/gwt/src/org/rstudio/studio/client/application/ui/AboutDialogMessages.java

```diff
@@ -0,0 +1,8 @@
+package org.rstudio.studio.client.application.ui;
+
+import com.google.gwt.i18n.client.Messages;
+
+public interface AboutDialogMessages extends Messages {
+    @DefaultMessage("|About {0}")
```

Comment:
What does the leading `|` do? I'm also completely new to Java/GWT loc.

Reply:
Sorry this is debugging code that I missed pulling out.  I find it really hard to tell if GWT is displaying the `@DefaultMessage` or the `_en.properties` text, so while debugging I sometimes prepend text on one or the other (same reason why I ended up adding the `dev` locale scripts).  I'll remove